pkg/kubectl/resource_printer.go

```diff
@@ -1021,32 +957,37 @@ func printIngress(ingress *extensions.Ingress, w io.Writer, options PrintOptions
 		return err
 	}
 
-	// Lay out all the rules on separate lines.
-	extraLinePrefix := ""
-	if options.WithNamespace {
-		extraLinePrefix = "\t"
-	}
-	for _, rules := range hostRules {
-		if rules.HTTP == nil {
-			continue
-		}
-		_, err := fmt.Fprintf(w, "%s\t%v\t", extraLinePrefix, rules.Host)
-		if err != nil {
-			return err
-		}
-		if _, err := fmt.Fprint(w, appendLabelTabs(options.ColumnLabels)); err != nil {
-			return err
+	// Lay out all the rules on separate lines if use wide output.
```

Comment:
I suggest not changing `kubectl get ingress` in this PR. We need the container/image fix in ASAP.

Ingress won't be compliant with the convention, but we can fix it in 1.3, if necessary.